templates/partials/replay/players.html

```diff
@@ -0,0 +1,5 @@
+<div>PLAYER 1</div>

```

Comment:
Why 1-5?

Reply:
This is just temporary for structure.
It will eventually be dynamically filled with players and their camera settings and loadout.